python/nx-cugraph/README.md

```diff
@@ -7,11 +7,10 @@ to run supported algorithms with GPU acceleration.
 ## System Requirements
 
 nx-cugraph requires the following:
-
- * NVIDIA GPU, Pascal architecture or later
+ * NVIDIA GPU, Volta architecture or later, with [compute capability](https://developer.nvidia.com/cuda-gpus) 7.0+
  * CUDA 11.2, 11.4, 11.5, 11.8, or 12.0
- * Python versions 3.9, 3.10, or 3.11
- * NetworkX >= version 3.2
+ * Python version 3.10 or 3.11
```

Comment:
Why drop Python 3.9? We still support Python 3.9, which is usable with older networkx versions. Even NetworkX 3.2 supports Python 3.9.

Reply:
Oops, you're right, that was an oversight.  Fixed.